pkg/sourcereader/local_test.go

```diff
@@ -115,13 +58,29 @@ func (s *MySuite) TestGetModule_Local(c *C) {
 
 	// Invalid: No local module
 	badLocalMod := "./modules/does/not/exist"
-	err = reader.GetModule(badLocalMod, dest)
+	err = s.r.GetModule(badLocalMod, dest)
 	expectedErr = "local module doesn't exist at .*"
 	c.Assert(err, ErrorMatches, expectedErr)
 
 	// Invalid: Unsupported Module Source by LocalSourceReader
 	badSource := "gcs::https://www.googleapis.com/storage/v1/GoogleCloudPlatform/hpc-toolkit/modules"
-	err = reader.GetModule(badSource, dest)
+	err = s.r.GetModule(badSource, dest)
 	expectedErr = "source is not valid: .*"
 	c.Assert(err, ErrorMatches, expectedErr)
 }
+
+func (s *localSuite) TestCopyFromPath_Local(c *C) {
```

Comment:
Do you test if the path to copy from doesn't exist?

Reply:
Good point. Added test for this